LayoutTests/http/wpt/screen-wake-lock/sticky-permission-after-transient-activation.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+promise_test(async t => {
+    // Does not have permission, should reject.
+    await promise_rejects_dom(t, "NotAllowedError", navigator.wakeLock.request('screen'));
+
+    // Trigger transient activation.
+    internals.withUserGesture(() => { });
+    let lock = await navigator.wakeLock.request('screen');
+
+    // Consume transient activation.
+    assert_true(internals.consumeTransientActivation());
+
+    await lock.release();
+
+    // This document was previously authorized so it should be able to require the wake lock again.
+    lock = await navigator.wakeLock.request('screen');
```

Comment:
Maybe overkill, but: 

```suggestion
    assert_true(navigator.userActivation.hasBeenActive, "has sticky activation");
    lock = await navigator.wakeLock.request('screen');
```